theme.js

```diff
@@ -125,6 +140,7 @@ export const theme = {
         'Monaco', 'DejaVu Sans Mono', 'Lucida Console'
       ]
     },
+    gridTemplateColumns: { ...Object.fromEntries(columns) },
```

Comment:
I believe you meant:
```suggestion
    gridTemplateColumns: { ...Object.fromEntries(autoColumns) },
```